docs/fsharp/index.md

```diff
@@ -44,15 +44,23 @@ To learn more and get involved, check out [fsharp.org](http://fsharp.org).
 
 ## Documentation
 
-* [Getting Started](tutorials/getting-started/index.md)
-* [Introduction to Functional Programming](introduction-to-functional-programming/index.md)
 * [Tutorials](tutorials/index.md)
+* [Introduction to Functional Programming](introduction-to-functional-programming/index.md)
 * [Language Reference](language-reference/index.md)
 * [F# Core Library Reference](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/fsharp-core-library-reference)
 
+## Online Learning Resources
```

Comment:
I have one question though: videos are also online learning resources. Maybe you could combine them or give a different section title here?